pkg/spec/util.go

```diff
@@ -65,12 +66,20 @@ func GetScheme() (*runtime.Scheme, error) {
 	// Initializing the scheme with the core v1 api
 	scheme := api.Scheme
 
+	// TODO: find a way where we don't have to add all the subsequent schemes
```

Comment:
Why different schema for different controllers? 
Using one with everything should be fine or not?

Reply:
It's fine, but it'd lead to a _huge_ schema. So this is more of an optimization TODO, does not really matter much right matter tbh.